packages/jest-cli/src/SearchSource.js

```diff
@@ -33,7 +33,7 @@ type Options = {|
   withAncestor?: boolean,
 |};
 
-export type PathPattern = {|
+export type PathPatternObject = {|
```

Comment:
we already have `testNamePattern` and `testPathPattern`, which is confusing. Also this object seem to have more info than just a pattern